src/consensus/vote.rs

```diff
@@ -54,6 +55,12 @@ pub enum Vote {
 
     // Voted to change the age of the given node.
     ChangeAge(MemberInfo),
+
+    // Voted to send user message on behalf of the whole section.
```

Comment:
bit comfused with `on behalf of the whole section`
thought the on concensus of the Vote, elders will send out the contained message, then that is already `whole` section doing so?

maybe just remove `on behalf of ...` to make the comment same pattern as others ?

Reply:
Yeah, you are right. It's badly worded. Will change it.